src/marlin_stubs/pause/pause.cpp

```diff
@@ -263,12 +263,20 @@ bool Pause::ensureSafeTemperatureNotifyProgress(uint8_t progress_min, uint8_t pr
         return true;
     }
 
-    setPhase(PhasesLoadUnload::WaitingTemp, progress_min);
+    setPhase(can_stop ? PhasesLoadUnload::WaitingTemp_stoppable : PhasesLoadUnload::WaitingTemp_unstoppable, progress_min);
```

Comment:
note:
this `can_stop ? ... : ...` is OK now
but we really should add transition table (and unify all state machines in FW)
it is my code, so it is not your fault 😃